hoodiehq/hoodie-cli
# recent dev work

- [tests] adds previously skipped tests
- [minor] adds TODO
- [fix] assert whether git is installed [#24]
- [minor] add grunt-mocha dependency
- [minor] clean up start path logic
- [wip] refactor tests
- introduce mocha for testing
- install grunt on travis bootstrap
- use ./node_modules/hoodie-server/bin/start if hoodie-app module not found
- check for hoodie-app existence before starting.
- no <3's for single letter vars
- minor refactoring and commenting
